data_contribution_2_data.md

```diff
@@ -0,0 +1,907 @@
+
+![Graphic representation of the lifecycle of the project template in Arcus](media/project_template/ProjectTemplate_Lifecycle.png)
+
+The project template provides a shared structure so that institutional knowledge previously held locally by various members of the data creation team becomes centralized.
+
+The utility of [the project template](https://github.research.chop.edu/arcus/rdm-project-template) for lab drive organization and integration with the Arcus archives is summarized in the graphic below.
+
+![Graphic representation of the workflow of research data and the project tempalte](media/project_template/ProjectTemplate_workflow.png)
+
+## Project Template
+
+The project template structure includes directories for capturing three major aspects of a research effort: the data (data), the tools needed to work with that data (access tools), and the contextual information needed to understand the effort and its constituent parts (contextual). The high level directories are as follows (items with asterix are required):
+
+- **_Configs_** (contextual)
+- **_Data_** (data)\*
+- **_Manifests_** (data)\*
+- **_Models_** (access tools)
+- **_References_** (contextual)
+- **_Reports_** (contextual)
+- **_Requirements_** (contextual)
+- **_SRC_** (access tools)
+
+Below is an image of the entire Project Template Directory, with more detail about each section:
+
+![Graphic respresentation of the Project template with a short explanation of all the different sections](media/project_template/ProjectTemplate_Description.png)
+
+### Research Data
+
+The Project Template brings together three categories of information: Research Data, Access Tools and Contextual files. Research data is the actual data collected during the course of research processes used for analysis. The manifests describe this data, crosswalking files to participants. Research data (with manifests) is the mininum required information for all Arcus data contributions.
+
+![Graphic respresentation of the Project template with that denotes research data sections](media/project_template/research_data.png)
+
+### Access Tools
+
+Access Tools are the code used to do the analysis. This can include machine models, scripts and Jupyter notebooks.
+
+![Graphic respresentation of the Project template with that denotes access tool sections](media/project_template/access_tools.png)
+
+### Contextual Files
+
+Contextual Files provide information needed to understand the data and analysis. This can include omics protocols, data dictionaries, reports and diagrams.
+
+![Graphic respresentation of the Project template with that denotes contextual files sections](media/project_template/contextual_files.png)
```

Comment:
Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.